redhat-cop/infra-ansible
# Support PVC storage for containerized Ansible Tower running on OCP

What does this PR do?
This PR extends the already present config-ansible-tower-ocp role with cability of automatic deployment of PVC for PostgreSQL persistent storage.

How should this be tested?
Please see test directory for instructions and test inventory

Is there a relevant Issue open for this?
N/A

Other Relevant info, PRs, etc.
N/A

People to notify
cc: @redhat-cop/infra-ansible
@oybed
